init.lua

```diff
@@ -19,9 +19,10 @@ local items = {
 	"       to trap team mates on purpose.",
 	"    b. Help the other team win.",
 	"    c. Change teams.",
-	"6. Don't impersonate other community members",
-	"7. Do not share your password with ANYONE.",
-	"8. Moderator decisions are final.",
+	"6. Don't combat-log, that is, leaving the game whilst in a fight",
```

Comment:
```suggestion
	"6. Don't leave the game whilst in a fight",
```

Reply:
Combat-logging is a technical term. It might be better to leave it out of this and keep it simple.